internal/packager/cnab.go

```diff
@@ -23,11 +23,24 @@ type DockerAppCustom struct {
 	Payload json.RawMessage `json:"payload,omitempty"`
 }
 
+// DockerAppArgs represent the object passed to the invocation image
+// by Docker App.
+type DockerAppArgs struct {
+	// Labels are the labels to add to containers on run
+	Labels map[string]string `json:"labels,omitempty"`
+}
+
 // ToCNAB creates a CNAB bundle from an app package
 func ToCNAB(app *types.App, invocationImageName string) (*bundle.Bundle, error) {
 	mapping := ExtractCNABParameterMapping(app.Parameters())
 	flatParameters := app.Parameters().Flatten()
 	definitions := definition.Definitions{
+		internal.ParameterArgs: {
+			Type:        "string",
+			Default:     "",
+			Title:       "Labels",
+			Description: "Labels to apply to service containers",
```

Comment:
Maybe we want to be less specific here since `DockerAppArgs` may contain other extra parameters than labels in the future?

Reply:
Nice catch! I started with a new parameter only for labels and then changed course and added a file, will change, thanks